src/pages/AllPhrases.js

```diff
@@ -3,49 +3,57 @@ import { Link } from 'react-router-dom';
 import ModalLoadingInfo from '../components/ModalLoadingInfo.js';
 import PhraseCard from '../components/PhraseCard.js';
 import AddIcon from '../icons/AddIcon.js';
+import FilterList from '../components/FilterList.js';
 
 export default function AllPhrases({
-  onBookmarkClick,
-  phrases,
-  onDeleteClick,
-  onUpload,
   cloudname,
-  onImageDeleteClick,
+  filterButtons,
   loadingStatus,
+  phrases,
   isLoading,
+  onBookmarkClick,
+  onDeleteClick,
+  onFilterClick,
+  onImageDeleteClick,
+  onUpload,
 }) {
   return (
     <AllPhrasesWrapper>
+      {isLoading ? <ModalLoadingInfo loadingStatus={loadingStatus} /> : null}
```

Comment:
You can achieve the same with this code:
```suggestion
      {isLoading && <ModalLoadingInfo loadingStatus={loadingStatus} />}
```